games/brickmatch/Kconfig

```diff
@@ -16,21 +16,21 @@ config GAMES_SHIFT
 		NOTE: The source code here is not based on that code from
 		above github.
 
-if GAMES_SHIFT
+if GAMES_BRICKMATCH
 
-config GAMES_SHIFT_PROGNAME
+config GAMES_BRICKMATCH_PROGNAME
 	string "Program name"
-	default "shift"
+	default "brick"
```

Comment:
`brick` -> `brickmatch` ? :-)

Reply:
It was on purpose, brickmatch is too big to type in the terminal. I thought using "bm" but it could mess with some other app in the future